content/community/gsoc-2023.it.md

```diff
@@ -0,0 +1,54 @@
+
+La XMPP Standards Foundation (XSF) intende partecipare alla [Google Summer of Code 2023](https://summerofcode.withgoogle.com/programs/2023) come organizzazione ospitante.
+Se siete interessati a partecipare come studenti (o semplici neofiti dell'open source), mentori o progetti XMPP, questo è il sito da cui partire!
+ 
+Se potete immaginare di lavorare part-time (o anche a tempo pieno) su un progetto di software open source durante l'estate, prendete in considerazione l'idea di unirvi a un progetto di comunicazione XMPP e di partecipare alla Google Summer of Code 2022!
+
+### XMPP - il protocollo estensibile di messaggistica e presenza
+
+XMPP è l'Extensible Messaging and Presence Protocol, un insieme di tecnologie aperte per la messaggistica istantanea, la presenza, le chat multi-party, le chiamate vocali e video, la collaborazione, il middleware leggero, la syndication di contenuti e l'instradamento generale di dati XML. XMPP è stato originariamente sviluppato nella comunità open source Jabber per fornire un'alternativa aperta e decentralizzata ai servizi di messaggistica istantanea chiusi dell'epoca.
+
+### Google Summer of Code 2022
+
+La Google Summer of Code è un programma internazionale annuale in cui Google assegna borse di studio ai partecipanti che completano con successo un progetto di codifica di software libero e open source durante l'estate. Le idee di progetto sono elencate dalle organizzazioni ospitanti impegnate nello sviluppo di software open source, ma gli studenti possono anche proporre le proprie idee di progetto. 
+
+### Progetti partecipanti
+
+XSF ha fatto domanda ed è stata accettata per partecipare alla Google Summer of Code 2023. Se siete interessati, leggete subito la nostra pagina designata sul Wiki XSF, vedi sotto.
+
+### Benvenuti nuovi collaboratori!
+
+I nostri nuovi collaboratori devono ancora essere annunciati! Sentitevi liberi di spargere la voce attraverso i nostri canali qui sotto!
+
+### Risorse e modi per partecipare
+
+Se sei un collaboratore o un progetto interessato, assicurati di leggere i primi due link! Il coinvolgimento e la comunicazione tempestiva con noi sono essenziali per una collaborazione proficua.
+
+- Leggete le risorse su [sito web di Google](https://summerofcode.withgoogle.com/help) e verificate se questo progetto è adatto a voi e ai vostri piani per quest'anno.
+- Date una prima occhiata alla pagina dei dettagli del nostro progetto XMPP sul [wiki XSF](https://wiki.xmpp.org/web/Google_Summer_of_Code_2023).
+- Potete facilmente raggiungerci [via web chat](https://xmpp.org/chat#converse/room?jid=gsoc@muc.xmpp.org) o consultare la pagina ["Getting started with XMPP"](https://xmpp.org/getting-started/) e raggiungerci tramite la nostra [chat pubblica](xmpp:gsoc@muc.xmpp.org?join).
+- Precedente [Archivio della partecipazione all'XSF Google Summer of Code](https://wiki.xmpp.org/web/GSoC#Overview)
+- Volantino per l'applicazione XMPP & Google Summer of Code 2023]()
```

Comment:
```suggestion
- Volantino per l'applicazione XMPP & Google Summer of Code 2023
```

Reply:
Leave, I will update soon

Reply:
Regardless, I would change the broken link until you change it ;)